apps/microtvm/zephyr/demo_runtime/src/main.c

```diff
@@ -248,6 +249,9 @@ void uart_irq_cb(const struct device* dev, void* user_data) {
 
 // Used to initialize the UART receiver.
 void uart_rx_init(struct ring_buf* rbuf, const struct device* dev) {
+  const struct uart_config config = {
+      .baudrate = UART_BAUDRATE, .parity = 0, .stop_bits = 1, .data_bits = 3, .flow_ctrl = 0};
```

Comment:
@mehrdadh Hi Mehrdad. I think it would be nice to use the enums from Zephyr here (for instance .data_bits  = 3 might be a bit enigmatic for some folks):

```
   const struct uart_config config = {
-      .baudrate = UART_BAUDRATE, .parity = 0, .stop_bits = 1, .data_bits = 3, .flow_ctrl = 0};
+      .baudrate = UART_BAUDRATE,
+      .parity = UART_CFG_PARITY_NONE,
+      .stop_bits = UART_CFG_STOP_BITS_1,
+      .data_bits = UART_CFG_DATA_BITS_8,
+      .flow_ctrl = UART_CFG_FLOW_CTRL_NONE
+  };
```

Reply:
I've tested your changes against the Disco board and it's alright. So aside the magic `25` in `TVM_CRT_MAX_PACKET_SIZE_BYTES+25` for the RX buffer which you're currently discussing with Andrew and seems to be determined experimentally (maybe leave a comment about it?) the change LGTM.

Reply:
I removed the UART setting since it didn't help with fixing the issue. Still trying to understand why magic number 25 works here.